docs/source/markdown/podman-container-clone.1.md

```diff
@@ -19,6 +19,10 @@ Set a custom name for the cloned container. The default if not specified is of t
 
 Remove the original container that we are cloning once used to mimic the configuration.
 
+#### **--pod**=*name*
+
+Clone the container in an existing pod. It is useful to move a container to an existing pod.
```

Comment:
This should document that the container will join the pod namespaces and thus loose it custom network configuration for example.